mock/overview.js

```diff
@@ -4,67 +4,67 @@ export const expirationLimit = 20;
 export const resultData = [
   {
     key: 1,
-    result: ['fio_2020.09.10T12.42.11', ' virtual506.virt.lab.eng.bos.redhat.com'],
+    result: ['mock result -1', ' virtual506.virt.lab.eng.bos.redhat.com'],
```

Comment:
Damage is done, but we should probably remove internal host names and replace them with "*.example.com" or something.